app/uk/gov/hmrc/soletraderidentificationfrontend/models/PageConfig.scala

```diff
@@ -18,7 +18,7 @@ package uk.gov.hmrc.soletraderidentificationfrontend.models
 
 import play.api.libs.json.{Json, OFormat}
 
-case class PageConfig(optServiceName: Option[String], deskProServiceId: String, signOutUrl: String)
+case class PageConfig(optServiceName: Option[String], deskProServiceId: String, signOutUrl: String, enableSautrCheck: Option[Boolean])
```

Comment:
```suggestion
case class PageConfig(optServiceName: Option[String], deskProServiceId: String, signOutUrl: String, enableSautrCheck:Boolean = false)
```

Sorry, can we change this to a boolean with a default parameter?